app/models/concerns/rad_company.rb

```diff
@@ -72,6 +75,34 @@ def usage_stats
     [usage_headers, usage_items, usage_data]
   end
 
+  def next_phone_number
```

Comment:
extracted from SP

Reply:
yep - FYI, in case you were wondering, it cycles from available phone numbers to avoid a limit on sending too many text messages from one number